src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -337,60 +352,100 @@ namespace dd
         if (!engineRead)
           {
 
-            int fixcode
-                = fixProto(this->_mlmodel._repo + "/" + "net_tensorRT.proto",
-                           this->_mlmodel._def);
-            switch (fixcode)
+            nvinfer1::INetworkDefinition *network = nullptr;
+            nvcaffeparser1::ICaffeParser *caffeParser = nullptr;
+            nvonnxparser::IParser *onnxParser = nullptr;
+
+            if (this->_mlmodel._model.find("net_tensorRT.proto")
```

Comment:
Do we want to put that into functions, e.g. `read_trt_caffe_model`, and same for onnx ?

Reply:
Done